src/components/compose.jsx

```diff
@@ -604,7 +604,15 @@ function Compose({
         const item = items[i];
         if (item.kind === 'file') {
           const file = item.getAsFile();
-          if (file && supportedMimeTypes.includes(file.type)) {
+          if (!file) {
+            alert(`Could not access the given attachment.`);
+            return;
+          }
+          else if (supportedMimeTypes !== undefined && !supportedMimeTypes.includes(file.type)) {
+            alert(`Your instance does not allow attachments of type "${file.type}".`);
```

Comment:
This also need i18n `t`. "File "[filename?] is not supported" would e better, I think? Using the file name if possible as mime types are kinda technical.

Actually, this whole alert code need to be outside of the loop as it'll show one alert for every invalid/unknown file 🤔. Probably need another variable to store the unattached files, then show alert like "Files ([filename, filename]) are not supported".